examples/server.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//! This example sets up a local Drawbridge server on port 12345.
+//! Run this example with `cargo run --example server`.
+
+use std::net::Ipv4Addr;
+
+use drawbridge_app::Builder;
+use hyper::Server;
+use tempfile::tempdir;
+
+#[tokio::main]
+async fn main() {
+    let store = tempdir().unwrap();
```

Comment:
I think we need to add support for persisting this and also take an already-existing directory in.